### ISSUE TITLE
Submit to distros package managers?

### ISSUE THREAD

**Issue by amgdev9** [darlinghq/darling#1558]
I think it would be useful to submit darling to popular distros official repositories (e.g. ubuntu/fedora repositories), this will simplify the installation process for the regular user while also building trust about the software

**Comment by Ardey0** [comment]
Agree! This would not only build trust but also help more people find out about darling.

**Comment by AliYar-Khan** [comment]
Agree and it will get some attention for contributions as well

**Comment by Matr1x-101** [comment]
It's already in the AUR afaik. I could probably help package it for Gentoo in GURU as I know how to write ebuilds. As for Fedora and Debian, you can submit an ITP at <https://www.debian.org/Bugs/>, and for Fedora it's at <https://bugzilla.redhat.com/>, anyone can do this (and I'm sure it would be appreciated!).
